File: create-icons.js
const sharp = require('sharp');
const fs = require('fs');

const svgContent = `<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 24 24" fill="#8B5CF6" stroke="#FFFFFF" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
  <polygon points="11 12 17 2 13 12 19 22 11 12"/>
  <polygon points="10 12 4 2 8 12 2 22 10 12"/>
</svg>`;

const sizes = [16, 48, 128];

async function createIcons() {
    for (const size of sizes) {
        await sharp(Buffer.from(svgContent))
            .resize(size, size)
            .toFile(`icon${size}.png`);
    }
}

createIcons().catch(console.error); 

File: create_icons.py
from cairosvg import svg2png

icon_svg = '''
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#7300ff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
    <path d="M4 14a1 1 0 0 1-.78-1.63l9.9-10.2a.5.5 0 0 1 .86.46l-1.92 6.02A1 1 0 0 0 13 10h7a1 1 0 0 1 .78 1.63l-9.9 10.2a.5.5 0 0 1-.86-.46l1.92-6.02A1 1 0 0 0 11 14z"/>
</svg>
'''

sizes = [16, 32, 48, 128]

for size in sizes:
    svg2png(bytestring=icon_svg,
            write_to=f'icons/icon{size}.png',
            output_width=size,
            output_height=size) 